src/app/donation-start/donation-start-form/donation-start-form-new.component.html

```diff
@@ -102,6 +101,53 @@
           of main donation
           </mat-hint>
         </div>
+        <app-donation-tipping-slider
+            *ngIf="!showCustomTipInput"
+            #donationTippingSlider
+            [percentageStart]="1"
+            [percentageEnd]="maximumTipPercentage"
+            [donationAmount]="donationAmount"
+            donationCurrency="GBP"
+            [onHandleMoved]="onDonationSliderMove"
+          />
+
+          <span class="button" (click)="toggleTipInput()" *ngIf="!showCustomTipInput && donationAmount">
+            Other amount
+          </span>
+
+          <span *ngIf="showCustomTipInput && tipAmount() && !(tipAmount() > donationAmount * tipPercentage)"  class="button" (click)="toggleTipInput()" >
+            Back to percentage tip
+          </span>
+
+        <mat-expansion-panel hideToggle>
+          <mat-expansion-panel-header>
+          </mat-expansion-panel-header>
+          <p>This is the primary content of the panel.</p>
+        </mat-expansion-panel>
+        <mat-expansion-panel (opened)="panelOpenState = true"
+                             (closed)="panelOpenState = false">
+          <mat-expansion-panel-header>
+            <h3 class="b-rt-0 b-m-0 b-bold">How does Big Give use tips?</h3>
+          </mat-expansion-panel-header>
+          <hr/>
+          <p>
+            Big Give is a registered charity (1136547). We don’t charge charities for using our platform; we rely on the generosity of people who donate to leave a Voluntary Tip, which helps us run our match funding campaigns.
+          </p>
+          <p>Your tips will be going to:</p>
+          <ul>
+            <li>Supporting our website and team, allowing charities to double their
+              donations to good causes.
+            </li>
+            <li>Providing match funds in our Anchor Match Fund. Through tips, charities will receive funding to launch their own match funded campaign on Big Give.
+            </li>
+          </ul>
+
+  <p>
+  Thank you for your generosity! Your tips ensure that we can continue to support charities and causes that are important to you.
+
+          </p>
```

Comment:
Is this paragraph meant to be here? Feels like its repeating the stuff above, could have been left in by mistake?